src/README.html

```diff
@@ -956,7 +956,7 @@ <h4 id="unicode-codepoints">
 
 <p>The <code translate="no" class="kw">bdi</code> element is used to ensure that example characters that are right-to-left do not interfere with the layout of the page. Do not include line breaks or a space between the closing <code translate="no" class="kw">bdi</code> and the following <code translate="no" class="kw">span</code> element; spacing and presentation is controlled by styling.</p>
 
-<p>The <code translate="no" class="kw">lang</code> attribute should be filled in appropriately to get the correct font selection for a given context. Examples in East Asian languages (such as Chinese, Japanese, or Korean) or in the Arabic script can sometimes require greater care in choosing a language tag.</p>
+<p>The <code translate="no" class="kw">lang</code> attribute should be filled in appropriately to get the correct font selection for a given context. Examples in East Asian languages (such as Chinese, Japanese, or Korean) or in the Arabic script can sometimes require greater care in choosing a language tag. In some cases, it might also be necessary for you to adjust the <kbd translate="no">font-family</kbd> and/or <kbd translate="no">font-size</kbd> in your own style sheet for characters or character sequences in a given language.</p>
```

Comment:
I think if might be useful to add "for the content of the bdi element" in there somewhere - just so that people don't style the .codepoint element.

In case it helps, the first para of the editorial guidelines now says:

> Use the following markup to represent a character. For certain languages, it may be necessary to style the bdi element with a font-family and font-size.